include/shell/shell_bt_nus.rst

```diff
@@ -13,66 +13,74 @@ Setup
 
 Enable NUS shell transport in your application to be able to receive shell commands remotely.
 
-To send shell commands to an application that uses this module, you need specific software, comparable to a terminal (for example, PuTTY) for UART communication.
+To send shell commands to an application that uses this module, you need a serial console application, like PuTTY, for UART communication.
 
 .. _shell_bt_nus_host_tools:
 
 Sending shell commands
 **********************
 
-The |NCS| provides two alternatives for sending shell commands from a host (for example, a PC) to the application that uses this module.
-`bt_nus_shell.py`_ is a Python script that requires a terminal and a second development board.
-`BLE Console`_ is a stand-alone application for Linux.
+.. testing_general_start
+
+The |NCS| provides two alternatives for sending shell commands from a host, like a PC, to the application that uses this module:
+
+* `bt_nus_shell.py`_. It is a Python script that requires a serial console application, like PuTTY, and a second development kit.
```

Comment:
i would remove `serial`, just `console application` since techincally in Putty we are not using serial communication but raw communication with network port.

Reply:
fixed